docs/source/access/accessing_cod.ipynb

```diff
@@ -0,0 +1,525 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<AxesSubplot:>"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# prove that we can read the AWS files\n",
+    "# for more information - https://automating-gis-processes.github.io/CSC/notebooks/L5/read-cogs.html\n",
+    "with rio.Env(aws_session):\n",
+    "    with rio.open(file_s3[1], 'r') as src:\n",
+    "        # list of overviews\n",
+    "        oviews = src.overviews(1)\n",
+    "        # get second item from list to retrieve a thumbnail\n",
+    "        oview = oviews[1]\n",
+    "        # read first band of file and set shape of new output array\n",
+    "        thumbnail = src.read(1, out_shape=(1, int(src.height // oview), int(src.width // oview)))\n",
+    "# now display the thumbnail\n",
+    "show(thumbnail)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since we verified that we can read the AWS files and display a thumbnail, we can create a mosaic from all of the rasters in our `file_s3` list. To do this, again pass the boto3 session to `rio.Env`. Then create a list which contains all of the read in Cloud Optimized Geotiffs (this may take a while)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create a mosaic from all the images\n",
+    "with rio.Env(aws_session):\n",
```

Comment:
This was pretty slow for me, perhaps our example should use fewer files?

Reply:
Sure we could update the bounding box to read from the corners of 4 files. But I'm also curious about how slow. It was tested on a pretty small workspace.

Reply:
Should probably pair so you can show us what's slow, it really shouldn't be.

Reply:
It might just be I need to restart my workspace. If it's faster for others I don't want this to be a blocking comment!

Reply:
We now cut the example to 4 items, and I added a GDAL env variable that skips trying to read side-car files and should make this faster. @spa0002 should be updating this PR with those changes.